src/helper_functions.jl

```diff
@@ -159,13 +159,13 @@ We check whether the matrix is square and whether the
 `cond(M)` is below some prescribed tolerance.
 """
 function isinvertible(M::Matrix; cond_tol::Number=DEFAULT_COND_TOL)
-    return issquare(M) && cond(M) < cond_tol
+    return cond(M) < cond_tol
```

Comment:
why? A non-square matrix cannot be invertible.

Reply:
Actually it is a good question how to fix it. 
The problem is when we are doing a projection of `HalfSpace` we have a sparse matrix, which is not a square by itself.

Reply:
I do not see the problem. If it is not square, then it is not invertible. End of the story. You cannot just change `isinvertible` and hope that this makes the matrix invertible :wink:

Reply:
Furthermore, this method is not used for sparse matrices.